Makefile

```diff
@@ -1,16 +1,18 @@
 .DEFAULT_GOAL := help
 
 # local-port:service-port
-CONDITION_API_PORT_FW=9001:9001
-CONDITION_ORC_API_PORT_FW=9002:9001
-ALLOY_PORT_FW=9091:9091
-FLEETDB_PORT_FW=8000:8000
-CRDB_UI_PORT_FW=8081:8080
-CRDB_PORT_FW=26257:26257
-CHAOS_DASH_PORT_FW=2333:2333
-JAEGER_DASH_PORT_FW=16686:16686
-OTEL_PORT_FW=4317:4317
-MINIO_PORT_FW=9000:9000
+
+WITH_LAN=
+CONDITION_API_PORT_FW=9001
+CONDITION_ORC_API_PORT_FW=9001
+ALLOY_PORT_FW=9091
+FLEETDB_PORT_FW=8000
+CRDB_UI_PORT_FW=8080
```

Comment:
Maybe we should keep CRDB UI as `8081:8080` since 8080 conflicts with FLEETDB.

Reply:
Looks like CRDB does use 8080:
https://www.cockroachlabs.com/docs/stable/ui-overview#db-console-access
And this was 8081 and then fixed to 8081:8080:
https://github.com/metal-toolbox/sandbox/commit/c1690d2016f27e8672a4f0c5af5d9e3092655ddd
So changing it to 8080 should be fine.